### ISSUE TITLE
WPEC RSS Feed Uses All Available Memory Often

### ISSUE THREAD

**Issue by JeffPyeBrook** [wp-e-commerce/WP-e-Commerce#2053]
On sites with not large product catalogs, but limited resources the WPEC rss feed logic will consume all PHP memory and throw a fatal error. The RSS feed logic should be adjusted to send the feed contents in "pages" so that pulling the entire product catalog and all of its meta into memory at the same time is not necessary.

**Comment by galbaras** [comment]
Yeah, just noticed it myself. I've tested up to 512MB of memory and it still bombs out.

This may be related to WP 4.9.2 or to Jetpack 5.7.1.

Is there a way to disable loading of some components during the feed generation?
